src/System.IO.Compression/src/System/IO/Compression/Crc32Helper.cs

```diff
@@ -450,14 +453,36 @@ internal static class Crc32Helper
             0x264B06E6u
         };
 
+        private static bool IsZlibCrcAvailable()
+        {
+            try
+            {
+                // Make a P/Invoke into zlib crc32 to ensure we're able to find and use it.
+                // If we are, then use zlib.
+                Interop.zlib.crc32(0, null, 0, 0);
```

Comment:
Seems like this could just be crc32(0, null, 0) and we wouldn't need the branch in the callee.